File: JAVA/Practica8/PROBLEMA_DIAMANTE/A.java
package JAVA.Practica8.PROBLEMA_DIAMANTE;

public class A extends C {
    public int x;
    public A(int x, int z) {
        super(z);
        this.x = x;
    }
    public void incrementaXZ() {
        this.x += 1;
        this.z += 1;
    }
    public void incrementaZ() {
        this.z += 1;
    }
}

File: PYTHON/Practica8/PROBLEMA_DIAMANTE/A.py
from C import C

class A(C):
    def __init__(self, x, z):
        C.__init__(self, z)
        self.x = x
    def incrementaXZ(self):
        self.x += 1
        self.z += 1
    def incrementaZ(self):
        self.z += 1

